packages/react/src/utils/createSpaceInternal.ts

```diff
@@ -0,0 +1,54 @@
+import { Flatfile } from '@flatfile/api'
+
+type CreateSpaceinternal = {
+  apiUrl: string
+  publishableKey: string
+  space: Flatfile.SpaceConfig
+  workbook?: Flatfile.CreateWorkbookConfig
+  document?: Flatfile.DocumentConfig
+}
+
+export const createSpaceInternal = async ({
+  apiUrl,
+  publishableKey,
+  space,
+  workbook,
+  document,
+}: CreateSpaceinternal) => {
+  const createSpaceEndpoint = `${apiUrl}/v1/internal/spaces/init?publishableKey=${publishableKey}`
+
+  let spaceRequestBody: any = {
+    space,
+  }
+
+  if (workbook) {
+    spaceRequestBody = {
+      ...spaceRequestBody,
+      workbook,
+    }
+  }
+
+  if (document) {
+    spaceRequestBody = {
+      ...spaceRequestBody,
+      document,
+    }
+  }
+
+  try {
+    const response = await fetch(createSpaceEndpoint, {
+      method: 'POST',
+      headers: {
+        Accept: 'text/plain',
```

Comment:
@HJordan35 I took this from the `@flatfile/javascript` package but Coderabbit is barking at this type. Is this some hackery needed for our BFF end point or can we change it to `application/json`

Reply:
I think it's actually on purpose to avoid a CORS request

Reply:
Makes sense, just wanted to confirm.